zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/Endpoint.kt

```diff
@@ -115,6 +116,8 @@ class Endpoint internal constructor(
     }
   }
 
+  private val functionsCache = mutableMapOf<String, List<ZiplineFunction<*>>>()
```

Comment:
Might want to use a LruCache?

Reply:
Fine for now. There’s no LruCache in Kotlin/JS!